src/rust/.cargo/config.toml

```diff
@@ -0,0 +1,11 @@
+[target.x86_64-apple-darwin]
```

Comment:
Why is this required? So we can build without going through `setuptools-rust`?

Reply:
It's not required. It  just makes it easier to test the build script locally with `cargo build`. I accidentally committed it. Will remove it later.